packages/kmod-5.15-nvidia/disable-gsp.service.in

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Disable GSP for selected instance types
+RequiresMountsFor=/etc/modprobe.d PREFIX/share/factory/etc/modprobe.d
+After=network-online.target
+Before=load-kernel-modules.service
+# Rerunning this service after the system is fully loaded will accomplish
+# nothing except write to a file that will never be used (after boot).
+# Refuse manual intervention.
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+ExecCondition=/usr/bin/shibaken does-value-start-with instance-type \
+  --value g4dn. --value g5. --value g5g.
```

Comment:
Is this needed on `g6` also?

Reply:
Probably. I'm not able to launch a g6 instance to see what nvidia_smi would tell me.